Assets/Scripts/Interactables/Dialogue/Conversation.cs

```diff
@@ -18,6 +18,20 @@ public class Conversation : ScriptableObject
     [Header("Note: Autoplay Conversation takes precedence over Auto Initiate")]
     public bool autoplayConversation = false;
     public bool autoInitiate = false;
-    public bool shouldFollow = false;
     public Line[] lines;
+
+    public List<string> GetAllSpeakers()
+    {
+        List<string> allSpeakers = new List<string>();
+
+        foreach(Line line in lines)
+        {
+            if(!allSpeakers.Contains(line.id))
+            {
+                allSpeakers.Add(line.id);
+            }
+        }
+
+        return allSpeakers;
+    }

```

Comment:
I'm an absolute sucker for one-liners sometimes so here's a one-line way to do it if you like:
(import this at the top)
using System.Linq;
...
```suggestion
    public List<string> GetAllSpeakers()
    {
        return lines.Select(line => line.id).Distinct().ToList();
    }

```